lsst-epo/epo-react-lib
# Serverside canvas render

JIRA: https://jira.lsstcorp.org/browse/EPO-8809

## What this change does

Streamlines the clientside rendering process so a single canvas is created instead of 3 stacked canvases, and creates a `skia-canvas` powered server renderer.

## Notes for reviewers

Review using the investigations client to test the server renderer, client changes can be tested in Storybook

Include an indication of how detailed a review you want on a 1-10 scale.

- 5 = "Please make sure there are no obvious errors and that you believe it does what it says it does"

## Testing

In Storybook
- verify the various permutations of the color tool work (including in Safari), you can also test right-click save/copy the resulting image since it is now a single canvas element

In Investigations
- see investigation PR

## Screenshots (optional)

### Before:

### After:
